mojave/items/guns/weapons/revolver.dm

```diff
@@ -39,6 +39,18 @@
 	fire_sound = 'mojave/sound/ms13weapons/gunsounds/caravan/caravan2.ogg'
 	w_class = WEIGHT_CLASS_BULKY
 	weapon_weight = WEAPON_HEAVY
+////////////////////////// revolvers////////////////
+/obj/item/gun/ballistic/revolver/ms13/derringer
+	name = "derringer"
+	desc = "A small and sneaky 2 shot pistol thats often concealed. This one's a bit fancy with some gold trim."
+	icon_state = "derringer"
+	inhand_icon_state = "derringer"
+	force = 5
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/ms13/derringer
+	fire_sound = 'mojave/sound/ms13weapons/22pistol.ogg'
+	w_class = WEIGHT_CLASS_SMALL
+	extra_damage = 30
```

Comment:
```suggestion
	extra_damage = 25
```

For general consistency with the fact this is still just a .22 weapon, damage shouldn't be **double** the other two .22 weapons.